AsyncDisplayKit/Layout/ASLayoutOptionsPrivate.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <AsyncDisplayKit/ASDisplayNode.h>
+#import <AsyncDisplayKit/ASLayoutSpec.h>
+
+
+@interface ASDisplayNode()
```

Comment:
We should consider moving these into the primary header files, although it's fine here for now.  I'm worried the existence of these properties is a bit hidden.  I haven't seen the ASDN header yet, though, which maybe enumerates all the properties in the primary header.